allocate.go

```diff
@@ -100,6 +100,12 @@ type ExecAllocator struct {
 	initFlags map[string]interface{}
 
 	wg sync.WaitGroup
+
+	stdout io.ReadCloser
+	stderr io.ReadCloser
```

Comment:
seems to me like these don't need to be fields? local vars should suffice

Reply:
Agreed and done.